--- a/common/optparser.c	2026-01-24 12:13:40.545763055 +0800
+++ b/common/optparser.c	2026-01-24 12:13:40.617763055 +0800
@@ -39,6 +39,14 @@
 #include <strings.h>
 #endif
 #include <ctype.h>
+#include <limits.h>
+
+/* When building with -std=gnu90, some toolchains may not provide LLONG_MAX
+ * even though 'long long' exists as an extension. */
+#ifndef LLONG_MAX
+#define LLONG_MAX __LONG_LONG_MAX__
+#endif
+
 
 // libclamav
 #include "clamav.h"
