--- a/reformatter/CMakeLists.txt	2026-01-27 14:20:16.353884239 +0800
+++ b/reformatter/CMakeLists.txt	2026-01-27 14:20:16.505884239 +0800
@@ -34,10 +34,9 @@
     TARGET_LINK_LIBRARIES(json_reformat m)
 ENDIF (NOT WIN32)
 
-# copy the binary into the output directory
-GET_TARGET_PROPERTY(binPath json_reformat LOCATION)
-
 ADD_CUSTOM_COMMAND(TARGET json_reformat POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${binPath} ${binDir})
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different
+            $<TARGET_FILE:json_reformat>
+            ${binDir}/$<TARGET_FILE_NAME:json_reformat>)
 
 INSTALL(TARGETS json_reformat RUNTIME DESTINATION bin)

--- a/verify/CMakeLists.txt	2026-01-27 14:20:16.365884239 +0800
+++ b/verify/CMakeLists.txt	2026-01-27 14:20:16.505884239 +0800
@@ -28,10 +28,9 @@
 
 TARGET_LINK_LIBRARIES(json_verify yajl_s)
 
-# copy in the binary
-GET_TARGET_PROPERTY(binPath json_verify LOCATION)
-
 ADD_CUSTOM_COMMAND(TARGET json_verify POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${binPath} ${binDir})
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different
+            $<TARGET_FILE:json_verify>
+            ${binDir}/$<TARGET_FILE_NAME:json_verify>)
 
 INSTALL(TARGETS json_verify RUNTIME DESTINATION bin)
