diff --git a/reformatter/CMakeLists.txt b/reformatter/CMakeLists.txt
index 1ea291f..c1a31c2 100644
--- a/reformatter/CMakeLists.txt
+++ b/reformatter/CMakeLists.txt
@@ -35,10 +35,11 @@ IF (NOT WIN32)
     TARGET_LINK_LIBRARIES(json_reformat m)
 ENDIF (NOT WIN32)
 
-# copy the binary into the output directory
-GET_TARGET_PROPERTY(binPath json_reformat LOCATION)
-
 ADD_CUSTOM_COMMAND(TARGET json_reformat POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${binPath} ${binDir})
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different
+            $<TARGET_FILE:json_reformat>
+            ${binDir}/$<TARGET_FILE_NAME:json_reformat>)
 
 INSTALL(TARGETS json_reformat RUNTIME DESTINATION bin)
diff --git a/verify/CMakeLists.txt b/verify/CMakeLists.txt
index db8b5d3..331f2fe 100644
--- a/verify/CMakeLists.txt
+++ b/verify/CMakeLists.txt
@@ -29,10 +29,11 @@ ADD_EXECUTABLE(json_verify ${SRCS})
 
 TARGET_LINK_LIBRARIES(json_verify yajl_s)
 
-# copy in the binary
-GET_TARGET_PROPERTY(binPath json_verify LOCATION)
-
 ADD_CUSTOM_COMMAND(TARGET json_verify POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${binPath} ${binDir})
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different
+            $<TARGET_FILE:json_verify>
+            ${binDir}/$<TARGET_FILE_NAME:json_verify>)
 
 INSTALL(TARGETS json_verify RUNTIME DESTINATION bin)
